ComfortZoneApp/app/src/main/java/com/example/comfortzone/utils/FilteringUtils.java

```diff
@@ -88,4 +86,8 @@ public int compare(WeatherData o1, WeatherData o2) {
         });
     }
 
+    public static List<WeatherData> searchCity(CharSequence name, List<WeatherData> cities) {
+        return cities.stream().filter(city -> city.getCity().toLowerCase(Locale.ROOT).contains(name)).collect(Collectors.toList());
```

Comment:
is name guaranteed to be lower case? I think we should do it here so it doesn't depend on input